tests/functional/adapter/test_unit_tests.py

```diff
@@ -32,16 +32,6 @@
 
 class TestTestingTypesTeradata(BaseUnitTestingTypes):
 
-    @pytest.fixture(scope="class")
```

Comment:
Why these tests are removed

Reply:
We haven't removed those test cases; instead, we have overridden and customized them, changing the materialization to "table" because unit tests for views are not supported. With these changes, the unit test for the view now works as expected, and there is no need for any further overrides or changes. We are now running the unit tests as they are from dbt-core.